moshi/src/test/java/com/squareup/moshi/CircularAdaptersTest.java

```diff
@@ -97,19 +100,20 @@ Node minusPrefix(String prefix) {
    * work.
    */
   static class PrefixingNodeFactory implements JsonAdapter.Factory {
-    @Override public JsonAdapter<?> create(Type type, AnnotatedElement annotations, Moshi moshi) {
+    @Override public JsonAdapter<?> create(
+        Type type, Set<? extends Annotation> annotations, Moshi moshi) {
       if (type != Node.class) return null;
 
       final String prefix;
-      if (annotations.isAnnotationPresent(Left.class)) {
+      if (Util.isAnnotationPresent(annotations, Left.class)) {
```

Comment:
Something to think about: this convenience method won't be available to those writing custom adapter factories.